make/mt-config/mt-help.mk

```diff
@@ -67,17 +67,20 @@ help_build_presets:
 	$(info . CONFIG and TARGET flags are logged to the console so that you know which )
 	$(info . options have been enabled for your build.)
 	$(info )
-	$(info . A List of Popular Valid Presets: )
+	$(info . Available presets: )
 	$(info .   POSIX_DEV_MIN)
 	$(info .   POSIX_DEV)
 	$(info .   POSIX_REL_MIN)
 	$(info .   POSIX_REL)
 	$(info .   POSIX_UNSECURE_REL)
 	$(info .   POSIX_THREADING_REL)
-	$(info .   CC3200_REL_MIN)
-	$(info .   CC3200_REL_MIN_UNSECURE)
-	$(info .   CC3200_REL)
-	$(info . )
+	$(info .   ARM_REL_MIN)
+	$(info .   ARM_REL)
+	$(info .   CC3200)
+	$(info .   CC3200_SDK120)
+	$(info .   CC3200_TLS_SOCKET)
+	$(info .   STM32F4)
+	$(info .   FUZZ_TESTS)
```

Comment:
I'm not sure if we should reveal FUZZ_TESTS for now... This is premature as this only works in certain conditions etc. I don't know. @DellaBitta what do you think ?